demos/advanced-dist/index.js

```diff
@@ -103,21 +105,21 @@ async function createPlayer (index, playerOptions) {
       }
     }
 
-    const iov = await iovCollection.create(videoElementId);
+    const clspIov = await clspIovCollection.create(videoElementId);
 
-    iov.changeSrc(url);
+    clspIov.changeSrc(url);
 
-    wallPlayers.push(iov);
+    wallPlayers.push(clspIov);
 
-    // iov.on('metric', (event, { metric }) => {
+    // clspIov.on('metric', (event, { metric }) => {
```

Comment:
@brainthinks Is this example code?

Reply:
it's the code used to enable metrics - i don't know if that feature works anymore, but the code for enabling it is still in there, commented out

Reply:
please delete

Reply:
deleted all metric code from demos